reporting_app/__init__.py

```diff
@@ -84,35 +84,42 @@ def change_password():
     return render_template('login.html', 'Login', message='Bad request.')
 
 
-@app.route('/pipelines/<pipeline_type>/<view_type>')
+@app.route('/runs/<view_type>')
 @flask_login.login_required
-def pipeline_report(pipeline_type, view_type):
-    statuses = {
-        'queued': ('reprocess', 'force_ready'),
-        'processing': ('processing',),
-        'finished': ('finished', 'failed'),
-        'archived': ('deleted', 'aborted')
-    }
-    endpoints = {'samples': 'aggregate/samples', 'runs': 'aggregate/all_runs'}
-    endpoint = endpoints[pipeline_type]
-
+def runs_report(view_type):
     if view_type == 'all':
-        query = rest_api().api_url(endpoint)
+        ajax_call = {
+            'func_name': 'merge_multi_sources',
```

Comment:
Are we wanting to keep this as is, or use `merge_multi_sources_keep_first`? (also below in view_type == 'recent')